package.json

```diff
@@ -41,6 +41,8 @@
     "debug": "^3.0.0",
     "depd": "^1.1.0",
     "dottie": "^2.0.0",
+    "env-cmd": "^5.0.0",
```

Comment:
`env-cmd` is a dev dependency, its already present in dev dependencies. Remove it from here

Reply:
I only aded the following dependency:

`"flat": "^4.0.0",`

But I'll remove it anyways.